### DISCUSSION TITLE
Ability to change fonts/theme UI for ePub in current ver?

### DISCUSSION THREAD

**Discussion by alexzmercury** [sumatrapdfreader/sumatrapdf#4089] [Q&A]
Hey!

I am currently running the latest release of SumatraPDF. I am reading an ePub and I'm stuck reading in Times New Roman with a white UI.

It seems like the application used to support changing the fonts in the advanced settings, but this is no longer. Is there any update or method of code that I can add which would allow me to read the **ePub documents in a different, specified font?**

Similarly, is there a way to **change the window UI colour to a different colour in advanced settings** aside from white? Upon changing the Darker theme, my pages change back to white, even having edited the page colour to a darker tone, so I have simply been editing the Light theme. I want a UI that is just very dark, including the page I am reading.

I saw similar threads, but it seems that they are in different vers, so I am unsure if there has been relevant updates since.

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/14340425/22ed78be-dfc9-4cde-9270-494abf00695a)

**Comment by kjk** [comment]
It's not currently possible to change the font size in epub reader.

**Reply by alexzmercury** [reply]
Thank you for that answer!

Is it possible to change the Font type? E.g., Arial, Helvetica, e.t.c.,?

**Reply by mianxiangdahai** [reply]
I also encountered the same problem. I hope that the new version of the software can customize and take effect different font types for the main text of epub files. Currently, only the font type of the directory can be customized, and the font type of the main text cannot be customized and take effect.

**Reply by trhorning** [reply]
> Thank you for that answer!
> 
> Is it possible to change the Font type? E.g., Arial, Helvetica, e.t.c.,?

By editing the stylesheet CSS inside the epub is the only way I have found to change it. It's a bit extreme but I read all my books with the same font so it works for me.
